mix.exs

```diff
@@ -64,10 +64,15 @@ defmodule JSONAPI.Mixfile do
         "Mitchell Henke",
         "Jake Robers",
         "Sean Callan",
-        "James Herdman"
+        "James Herdman",
+        "Mathew Polzin"
       ],
       licenses: ["MIT"],
-      links: %{github: "https://github.com/jeregrine/jsonapi", docs: "http://hexdocs.pm/jsonapi/"}
+      files: ["lib", "mix.exs", "README*", "LICENSE*"],
+      links: %{
+        github: "https://github.com/beam-community/jsonapi",
```

Comment:
Updated the GitHub URL now that this has been transferred into the beam-community org.